### DISCUSSION TITLE
[10->11] How can I upgrade custom Modules

### DISCUSSION THREAD

**Discussion by Vandal593** [OCA/OpenUpgrade#3815] [Q&A] [answered]
Hi, I am upgrading Odoo 10 to Odoo11, but I have a question about custom modules, I have some custom modules, but after upgrading Odoo some things are not working, how can I upgrade these modules, and is the upgrade only to the database or can I upgrade the code also?

**Comment by pedrobaeza** [comment, marked as answer]
OpenUpgrade only update database definitions, not code. You must update your custom module code, and maybe provide extra OpenUpgrade scripts for adapting data provided by your module.
